packages/react-query/src/shared/proxy/utilsProxy.ts

```diff
@@ -296,6 +301,8 @@ export const getQueryType = (
     case 'setInfiniteData':
       return 'infinite';
 
+    case 'setMutationDefaults': // This sounds wrong
```

Comment:
Not sure about this

Reply:
It should be good. When the input of `getQueryKey` is a mutation procedure the `QueryType` used is `"any"`